sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.26.3 (Unreleased)
+## 1.26.3 (2023-02-02)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Fixed deflate decompression for aiohttp   #28483
```

Comment:
nit: will the heading for this be bug fix ?

Reply:
Good catch. It was an accident. :)